pkg/apis/workloads/v1alpha1/console_types.go

```diff
@@ -20,9 +20,19 @@ type Console struct {
 
 // ConsoleSpec defines the specification for a console
 type ConsoleSpec struct {
-	User               string                      `json:"user"`
-	Reason             string                      `json:"reason"`
-	TimeoutSeconds     int                         `json:"timeoutSeconds"`
+	User   string `json:"user"`
+	Reason string `json:"reason"`
+
+	// Number of seconds that the console should run for.
+	// If the process running within the console has not exited before this
+	// timeout is reached, then the console will be terminated.
+	// Creation of a Console will be rejected if the specified timeout exceeds
+	// the maximum timeout defined within the ConsoleTemplate that the console is
+	// referring to.
+	// +kubebuilder:validation:Minimum=1
```

Comment:
Any reason to disallow 0? 1 and 0 both seem pretty silly to me, so I think if we allow the former we should allow the latter.